src/pages/SparkleSpaceMarketingPage/WelcomePage/WelcomePage.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from "react";
 import firebase from "firebase/app";
-import { useFirestoreConnect } from "react-redux-firebase";
+import { useSparkleFirestoreConnect } from "hooks/useSparkleFirestoreConnect";
 import { IFRAME_ALLOW, PLAYA_VENUE_NAME } from "settings";
 
 const WelcomePage: React.FunctionComponent = () => {
-  useFirestoreConnect("marketingemails");
+  useSparkleFirestoreConnect("marketingemails");
```

Comment:
This looks like a legacy. I didn't try to refactor it since it is not related to this PR. But probably there should be created a ticket to refactor this

Reply:
It might be another thing that can be removed as part of https://github.com/sparkletown/sparkle/pull/833 ?